src/collections/members/krishna-gupta/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+name: Krishna Gupta
+position: Contributor
+image_path: ./display-picture.png
```

Comment:
The failing build is likely due to this missing image.

Reply:
Ya I will add it when @Krishna2323 gives the image and bio